pkg/cmd/cli/cmd/newapp.go

```diff
@@ -684,6 +685,20 @@ func transformError(err error, c *cobra.Command, fullName string, groups errorGr
 			t.Errs...,
 		)
 		return
+	case newapp.ErrNoTagsFound:
+		buf := &bytes.Buffer{}
+		fmt.Fprintf(buf, "  Use --allow-missing-imagestream-tags to use this image stream\n\n")
```

Comment:
and if you have no tags on the image stream, then you're really saying you want to use the image stream anyway.